guide-configs/self-hosted/data-encryption.json

```
@@ -1,12 +1,12 @@
 {
     "file_name": "data-encryption.mdx",
     "id": "data-encryption",
-    "file_dir": "production-deployment/self-hosted-guide",
-    "slug": "/self-hosted-guide/data-encryption",
-    "title": "Encrypting application data",
-    "sidebar_label": "Data encryption",
-    "sidebar_position": 7,
-    "description": "This guide explains what you need to develop to use Temporal in your production environment.",
+    "file_dir": "production-deployment",
+    "slug": "/production-deployment/data-encryption",
+    "title": "Using a Codec Server",
```

```suggestion
    "title": "Codec Server - Temporal Platform feature guide",
```

This is the direction I want use to go to standardize on feature guide page titles.